.cirrus.yml

```diff
@@ -1,3 +1,35 @@
+task:
+  auto_cancellation: true
+
+  env:
+    matrix:
+      PY_VER: "3.6"
```

Comment:
do we really need to check against multiple python versions?

Reply:
maybe not